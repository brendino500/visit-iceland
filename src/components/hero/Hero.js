import LandingPageText from './LandingPageText'

export default function Hero() {
  return (
    <div>
      <img
        className="bject-none object-left-top absolute z-10 w-3/5"
        src="https://i.ibb.co/vzq9dCS/iceland19-min.jpg"
        alt="iceland19-min"
      />
      <LandingPageText />
    </div>
  )
}
