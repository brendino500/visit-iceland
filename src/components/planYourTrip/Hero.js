import { Typography } from '@material-ui/core'
import { makeStyles } from '@material-ui/core/styles'

const useStyles = makeStyles((theme) => ({
  content: {
    display: 'flex',
    justifyContent: 'center',
    alignItems: 'center',
    marginTop: '6em',
  },
  textContainer: {
    maxWidth: '35%',
    marginRight: '2em',
    fontFamily: 'Barlow',
    fontSize: '1.6em',
    textTransform: 'uppercase',
    letterSpacing: 1,
  },
  title: {
    fontFamily: 'Barlow',
    fontSize: '2.2em',
    textTransform: 'uppercase',
    textAlign: 'center',
    fontWeight: 'bold',
    letterSpacing: 2,
  },
  image: {
    height: '25em',
  },
}))

export default function Hero() {
  const classes = useStyles()
  return (
    <div>
      <Typography className={classes.title}>Plan your trip</Typography>
      <div className={classes.content}>
        <p className={classes.textContainer}>
          Iceland is a country of extreme contrasts where long summer days with
          near 24 hours of sunlight are offset by short winder days
        </p>

        <img
          src="https://i.ibb.co/jyQGK0F/iceland6-min.jpg"
          alt="iceland6-min"
          border="0"
          className={classes.image}
        />
      </div>
    </div>
  )
}
