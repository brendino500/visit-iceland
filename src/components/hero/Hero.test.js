import renderer from 'react-test-renderer'
import Hero from './Hero'

it('Hero matches snapshot', () => {
  const tree = renderer.create(<Hero />).toJSON()
  expect(tree).toMatchSnapshot()
})
