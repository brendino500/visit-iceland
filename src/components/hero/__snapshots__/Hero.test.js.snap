// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Hero matches snapshot 1`] = `
<div>
  <img
    alt="iceland19-min"
    className="bject-none object-left-top absolute z-10 w-3/5"
    src="https://i.ibb.co/vzq9dCS/iceland19-min.jpg"
  />
  <div
    className="container mx-auto flex content-center justify-end "
  >
    <div
      className="z-40"
    >
      <h1
        className="text-7xl uppercase text-gray-300 font-serif content-center mt-48 mr-80 "
      >
        Visit Iceland
      </h1>
    </div>
  </div>
</div>
`;
